internal/verifier/verifier.go

```diff
@@ -28,16 +28,15 @@ import (
 // ProvenanceVerifier defines an interface with a single method `Verify` for
 // verifying provenances.
 type ProvenanceVerifier interface {
-	// Verify verifies an Amber/SLSA provenance file in the given path.
```

Comment:
Why is the comment removed? Is not it the convention to have a doc comment for these methods?

Reply:
Fixed, thank you!